src/Esfa.Vacancy.Domain/Validation/ErrorMessages.cs

```diff
@@ -22,6 +22,8 @@ public static class SearchApprenticeships
             public const string GeoSearchAndNationwideNotAllowed = "Searching by geo-location and national vacancies is not a valid combination.";
 
             public const string SortByDistanceOnlyWhenGeoSearch = "You can only sort by distance if you have searched by geo-location.";
+
+            public static string SortByValueNotAllowed(string value) => $"'{value}' is not a valid value for 'Sort By', expected 'Age', 'Distance' or 'ExpectedStartDate'.";
```

Comment:
This message could get out of sync if we add more sortable fields. I think we should just say the field is invalid, the documentation defines the valid values.

Reply:
Updated